README.md

```diff
@@ -149,12 +146,26 @@ It's easiest to see these interactions visually:
 
 <img src="http://g.recordit.co/4ViVmKtKAr.gif">
 
-If your code starts getting cluttered up with results, run "Hydrogen: Clear Results" to remove them all at once. You can also run this command with ⌘-⌥-⌫.
+**"Hydrogen: Run And Move Down"** will run the the code as described above and move the cursor to the next executable line.
+
+If your code starts getting cluttered up with results, run **"Hydrogen: Clear Results"** to remove them all at once. You can also run this command with ⌘-⌥-⌫.
+
+### "Hydrogen: Run Cell"
+A "code cell" is a block of lines to be executed at once. You can define them within inline comments. Hydrogen supports a 
```

Comment:
s/You can define them **within** inline comments/You can define them **using** inline comments/